File: code.py
class Estabelecimento():
    
    def __init__(self, nome, endereco, categoria):
        self.nome = nome
        self.endereco = endereco
        self.categoria = categoria
        self.lista = []
        
        
class Gestor():
    
    def __init__(self, nome, cpf, email):
        self.nome = nome
        self.cpf = cpf
        self.email = email
        
class Camera():
    
    def __init__(self):
        self.ip = 0
        self.user = 0
        self.password = 0
        self.modelo = ''
        self.data_de_instalacao = 0
        self.contagem = {} # lista [[Data][Contagem]]
        
    def Insert(self, lista):
        lista_string=[]
        lista_itens = lista
        
        for i in lista_itens:
            i = i.split('=') #info[0].EndTime=2021-05-01 23:59:59
            dados = i[1]
            
            if 'EndTime' in i[0]:    #Data_de_Entrada
                lista_string.append(['EndTime',dados])
                
            elif 'camera' in i[0]:   #Qtd Entrada
                lista_string.append(['camera',dados])
                
            elif 'EnteredSubtotal' in i[0]:   #Qtd Entrada
                lista_string.append(['Entered',dados])
                
            elif 'ExitedSubtotal' in i[0]:   #Qtd Saída
                lista_string.append(['Exited',dados])
                
            elif 'StartTime' in i[0]:   #Data_de_Saída
                lista_string.append(['StartTime',dados])
                
        self.contagem[camera] = lista_string[0:]
    
    def printar(self):
        for k, v in self.contagem.items():
            for j in v:
                print(f'{j[0]} : {j[1]}')
                
    def printar_totais(self):
        soma = 0
        cont = 0
        lista = []
        #print(f'Câmera : {self.modelo}')
        for k, v in self.contagem.items():
            for j in v:
                if 'Entered' in j or 'Exited' in j:
                    soma += int(j[1])
                    if 'Exited' in j:
                        lista.append('Entradas Totais : '+ str(soma))
                else:
                    lista.append(f'{j[0]} : {j[1]}')
        
        print('')
        for string in lista:
            
            if 'camera' in string:
                cont += 1
                if cont > 1:
                    print('')
                    
            print(string)
        
        

lista = [] #lista de informações dos dados

prompt = input().split('=')

for f in range (0, int(prompt[1])):


    prompt_camera =  input('Camera: ')
    lista.append('camera=' + prompt_camera)

    for i in range(0,6):  #adicionando na lista os dados
        lista.append(input('Dados: '))


camera = Camera()
camera.Insert(lista)
camera.printar_totais()
    

File: code.js
class Estabelecimento {

    constructor(nome, endereco, categoria) {
        this.nome = nome;
        this.endereco = endereco;
        this.categoria = categoria;
    }

}

class Gestor {

    constructor(nome, cpf, email) {
        this.nome = nome;;
        this.cpf = cpf;
        this.email = email;
    }

}

class Camera {

    constructor(){
        var ip = 0;
        var user = 0;
        var password = 0;
        var modelo = modelo;
        var data_de_instalacao = 0;
        this.contagem = new Object();
    }


    Insert(lista) {
        var lista_string = [];
        var lista_itens = lista;

        lista_itens.forEach(element => {
            element = element.split('=');
            var dados = element[1]; //info[0].EndTime=2021-05-01 23:59:59

            var EndTime = element[0].includes('EndTime'); //True or False
            var camera = element[0].includes('camera');
            var EnteredSubtotal = element[0].includes('EnteredSubtotal');
            var ExitedSubtotal = element[0].includes('ExitedSubtotal');
            var StartTime = element[0].includes('StartTime');

            if (EndTime) {
                lista_string.push(['EndTime', dados]);
            }

            else if (camera) {
                lista_string.push(['camera',dados]);
            }

            else if (EnteredSubtotal) {
                lista_string.push(['Entered',dados]);
            }

            else if (ExitedSubtotal) {
                lista_string.push(['Exited',dados]);
            }

            else if (StartTime) {
                lista_string.push(['StartTime',dados]);
            }
        
           
        });
        this.contagem[camera] = lista_string; 
    }

    Printar() {
        var soma = 0;
        var cont = 0;
        var lista = JSON.parse(this.contagem);

        console.log(lista);

        /*for (let i of Object.values(this.contagem)) {
            lista = i;
            console.log(typeof(lista));
        }

        for (let [k,v] of Object.entries(this.contagem)) {
            for (let j in v) {
                console.log(j);
            }
        }*/
    }

    /*function printar_totais () {

    }*/

}

const { interfaces } = require("mocha");
var dado = '';
var lista = [];

const prompt = require("prompt-sync")();

var qtd_dados = prompt('Repetições de Dados: ').split('='); //Quantidade de contagem de dados

for (let i = 1; i <= parseInt(qtd_dados[1]); i++) {

    console.log('')
    var camera = prompt('Identificação da Câmera: ');
    lista.push('camera=' + camera); 

    for (let i = 0; i <= 5; i++) {
        dado = prompt('Dado: ').trim(); //lista de informações de dados
        lista.push(dado);
        
    }
}

var camera = new Camera();
camera.Insert(lista);
camera.Printar();

    
//var dois = prompt('teu: ');